config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,39 @@
+require 'carrierwave/orm/activerecord'
+
+case Rails.env
+  when 'test'
+    CarrierWave.configure do |config|
+      config.root              = Rails.root.join('tmp')
+      config.storage           = :file
+      config.enable_processing = false
+    end
+  else
+
+    if Settings.store_images_in_file_system
+
+      CarrierWave.configure do |config|
```

Comment:
Can we extract the whole block into a method called `setup_for_storing_images_in_file_system ?

I'm find it hard to follow logic with so much code inside a single case statement.